eclipse/che
# Add role for metrics

Signed-off-by: svor <vsvydenk@redhat.com>

### What does this PR do?

Adds new role **workspace-metrics** for workspace ServiceAccount to get an information about workspace's resources from Metric server. It makes possible to read metrics of the workspace which is running in different namespace than che.
 
### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/18812

### How to test this PR?
To check this changes, you need: 
1. deploy che with custom `cheimage` with current changes
2. try to create a workspace and check that it was created in the namespace `username-che` which is different than the namespes where che was deployed
3. try to get roles for `username-che` namespace

As a result `workspace-metrics` should be existed in the list

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
